components/organisms/SuggestedRepoList/suggested-repo-list.tsx

```diff
@@ -22,7 +22,7 @@ const SuggestedRepositoriesList = ({
   isLoading,
 }: SuggestedRepositoriesListProps) => {
   return (
-    <div>
+    <ClientOnly>
```

Comment:
refactor (blocking): The <ClientOnly /> component is preventing the hydration error now, but the `<div />` has been removed. It should remain to ensure no visual regressions.

The `<ClientOnly />` can be moved further down in the component so that we render as much markup sever-side still but laser focus on the dynamic part causing the hydration error.

e.g.

```
    <div>
      <Title className="!text-light-slate-11 !text-sm" level={4}>
        Suggested Repositories:
      </Title>

      <div className="flex flex-col gap-3 mt-6">
        <ClientOnly>
          {isLoading ? (
            <>
              <SkeletonWrapper count={3} classNames="w-3/4" height={60} radius={10} />
            </>
          ) : (
            <>
              {reposData.map((item, index) => (
                <SuggestedRepository key={index} data={item} loadingData={loadingData} onAddRepo={onAddRepo} />
              ))}
            </>
          )}
        </ClientOnly>
      </div>
    </div>
```